hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOmMetrics.java

```diff
@@ -442,64 +453,59 @@ private void testAclMetricsInternal(ObjectStore objectStore, OzoneObj volObj,
   /**
    * Test volume operations with ignoring thrown exception.
    */
-  private void doVolumeOps() {
+  private void doVolumeOps(OmVolumeArgs volumeArgs) {
     try {
-      ozoneManager.createVolume(null);
+      writeClient.createVolume(volumeArgs);
     } catch (IOException ignored) {
     }
 
     try {
-      ozoneManager.deleteVolume(null);
+      ozoneManager.getVolumeInfo(volumeArgs.getVolume());
     } catch (IOException ignored) {
     }
 
     try {
-      ozoneManager.getVolumeInfo(null);
+      writeClient.setOwner(volumeArgs.getVolume(), "dummy");
     } catch (IOException ignored) {
     }
 
     try {
-      ozoneManager.checkVolumeAccess(null, null);
+      ozoneManager.listAllVolumes("", null, 0);
     } catch (IOException ignored) {
     }
 
     try {
-      ozoneManager.setOwner(null, null);
-    } catch (IOException ignored) {
-    }
-
-    try {
-      ozoneManager.listAllVolumes(null, null, 0);
+      writeClient.deleteVolume(volumeArgs.getVolume());
     } catch (IOException ignored) {
     }
   }
 
   /**
    * Test bucket operations with ignoring thrown exception.
    */
-  private void doBucketOps() {
+  private void doBucketOps(OmBucketInfo info) {
```

Comment:
I also restructured this method to match the counts, by moving the delete to the end.  The diffs here make the changes look more significant than they here.  They really look like this:
|original                |current                 |
|------------------------|------------------------|
| createBucket            | createBucket            |
| deleteBucket            | [moved to end]                        |
| getBucketInfo           | getBucketInfo           |
| setBucketProperty                | setBucketProperty                |
| listBuckets          | listBuckets          |
|                        |deleteBucket            |